pyanaconda/ui/__init__.py

```diff
@@ -147,13 +153,21 @@ def _collectActionClasses(self, module_pattern_w_path, standalone_class):
         standalones = []
 
         for module_pattern, path in module_pattern_w_path:
-            standalones.extend(collect(module_pattern, path, lambda obj: issubclass(obj, standalone_class) and \
-                                       getattr(obj, "preForHub", False) or getattr(obj, "postForHub", False)))
+            standalones.extend(
+                collect(module_pattern,
+                        path,
+                        lambda obj: issubclass(obj, standalone_class) and
```

Comment:
Sometimes it is better to define a new function (it can be nested) instead of of using lambda functions.

Reply:
I've only reformatted the code not change it. But yeah why not.